installers/MacInstaller/make_package.rb

```diff
@@ -590,12 +590,13 @@ def stop_if_bundle_not_self_contained(basepath, extra = [])
   requirements_files << REQUIREMENTS_WORKBENCH_FILE
   bundled_qt_plugins = QT_PLUGINS_COMMON + ['platforms', 'printsupport', 'styles']
   host_qt_plugins_dir = QT5_PLUGINS_DIR
-  executables << "#{contents_macos}/MantidWorkbench"
+  executables << "#{contents_macos}/#{bundle_path.basename}"
+  fatal("MantidWorkbechUnstable? #{contents_macos}/#{bundle_path.basename}")
```

Comment:
This doesn't matter for the release but it will mean we can't build packages on pull requests. Does it currently not work in this case?

Reply:
I added this line strictly for debugging and will remove it soon.

I downloaded the a package from the build server and found that `MantidWorkbenchUnstable` is still linked against boost libraries in `/usr/local/`. Want to verify `#{contents_macos}/#{bundle_path.basename}` is giving what I expect before investigating further.